app/players/elves/elf.py

```diff
@@ -0,0 +1,14 @@
+from app.players.player import (Player)

```

Comment:
there are no parentheses in other modules, but here there are

Reply:
But we use parental nickname in these class

Reply:
Understood you, fixed in last commit, thanks